### PR TITLE
Add getting started with django

### PR DESCRIPTION
**Pull request by tychota**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/django/backend/django/getting-started.mo.md">getting-started.mo.md</a>**: MO doesn't have an *Owner*, could it be you?
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/django/backend/django/getting-started.mo.md">getting-started.mo.md</a>**: MO doesn't have a *Steps* part, how could you call this an MO?
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/django/backend/django/getting-started.mo.md">getting-started.mo.md</a>**: MO doesn't contain a *Prerequisites* part are you sure your brain is all you need?
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/django/backend/django/getting-started.mo.md">getting-started.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/django/backend/django/getting-started.mo.md">getting-started.mo.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: backend/django/getting-started.mo.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
+# [MO] Get Started with Django, in 3/4 hour (~45 min)
+
+# Owner: [Tycho Tatitscheff](https://github.com/tychota)
+
+# Prerequisites (~10 min)
+
+- Have **HomeBrew** installed (`ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)`) (~3 min)
```
Comment (yleflour) [inline backend/django/getting-started.mo.md]: Checkboxes

File: backend/django/getting-started.mo.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+    'django.contrib.staticfiles',
+    'mab.library'
+]
+```
+- Go back to root of the project: `cd ~/Code/django_formation`
+- Migrate the database: `python manage.py migrate`
+> **NOTE**: In order to keep the method of operation simple, we use the built in sqlite adapters. You can read more to change it the [beautiful documentation](https://docs.djangoproject.com/en/1.11/ref/settings/#databases).
+
+> **CHECK**:
+> - Create the django super user: `python manage.py createsuperuser` and fill with a username and a `very-secret(™)` password
```
Comment (yleflour) [inline backend/django/getting-started.mo.md]: This is not a check, a check is the ability to log in

File: backend/django/getting-started.mo.md [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+> - Run the server: `python manage.py runserver`
+> - Access `http://127.0.0.1:8000` and see the "It worked page"
+> - Access `http://127.0.0.1:8000/admin`, connect and see the admin page
+
+## Add the first model (~5 min)
+
+> **NOTE**: for editing Python code, I recommand using [Pycharm](insertLink). The debugging and auto complete features helps a lot. There is a community edition that is free to use.
```
Comment (yleflour) [inline backend/django/getting-started.mo.md]: How to setup VSCode for Python is surely coming :)

File: backend/django/getting-started.mo.md [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+```
+
+> **NOTE**: it is a good practice to add created and modifed fields to models, and add a verbose name.
+
+
+- Create auto-magicaly the migrations: `python manage.py makemigrations`
+- Migrate the database: `python manage.py migrate`
+
+> **CHECK 1**:
+> - Launch `python manage.py showmigrations`
```
Comment (yleflour) [inline backend/django/getting-started.mo.md]: Output?

**Pull request merged**